ajax/libs/react-sortable-hoc/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "react-sortable-hoc",
+  "filename": "react-sortable-hoc.min.js",
+  "version": "0.6.8",
+  "description": "Set of higher-order components to turn any list into a sortable, touch-friendly, animated list",
+  "author": {
+    "name": "Clauderic Demers",
+    "email": "me@ced.io"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/clauderic/react-sortable-hoc.git"
+  },
+  "keywords": [
+    "react",
+    "reactjs",
+    "react-component",
+    "sortable",
+    "sortable-list",
+    "list",
+    "sortable list",
+    "smooth",
+    "animated",
+    "hoc",
+    "higher-order",
+    "component"
```

Comment:
Let's add more keywords to help people search this lib easier.
![2018-02-19 9 23 21](https://user-images.githubusercontent.com/8349587/36359530-49a118e2-1557-11e8-9138-7109d8dee9f5.png)
Many thanks